packages/memory/tsup.config.js

```diff
@@ -0,0 +1,9 @@
+import { defineConfig } from 'tsup';
```

Comment:
let's do everything in the cli, so `tsup src/index.ts --format esm --dts --clean --treeshake`